linalg_ops/iree-e2e-attention-test.cc

```diff
@@ -208,10 +213,10 @@ static iree_status_t attention_results_initialize(
   out_results->value_elem_type = iree_hal_buffer_view_element_type(value);
   out_results->result_elem_type = iree_hal_buffer_view_element_type(result);
 
-  iree_hal_buffer_t* query_buffer = iree_hal_buffer_view_buffer(query);
-  iree_hal_buffer_t* key_buffer = iree_hal_buffer_view_buffer(key);
-  iree_hal_buffer_t* value_buffer = iree_hal_buffer_view_buffer(value);
-  iree_hal_buffer_t* result_buffer = iree_hal_buffer_view_buffer(result);
+  iree_hal_buffer_t *query_buffer = iree_hal_buffer_view_buffer(query);
+  iree_hal_buffer_t *key_buffer = iree_hal_buffer_view_buffer(key);
+  iree_hal_buffer_t *value_buffer = iree_hal_buffer_view_buffer(value);
+  iree_hal_buffer_t *result_buffer = iree_hal_buffer_view_buffer(result);
```

Comment:
move `*` characters back to the type, not the variable name? clang-format will follow the dominant style in the file, so you may need to change a few